editoast/src/views/timetable.rs

```diff
@@ -1226,7 +1226,7 @@ mod tests {
             .bytes();
         assert_eq!(
             &String::from_utf8(response).unwrap(),
-            "Failed to deserialize the JSON body into the target type: [0]: Duplicate exception key: 'duplicated_key_1' at line 1 column 2452"
+            "Failed to deserialize the JSON body into the target type: [0]: Duplicate exception key: 'duplicated_key_1' at line 1 column 2331"
```

Comment:
😓...

```suggestion
            String::from_utf8(response).unwrap().contains("Duplicate exception key: 'duplicated_key_1'")
```

to avoid hitting the same issue next time